File: Array/WaveArray/WaveArray.py
class Solution:
	# @param A : list of integers
	# @return a list of integers
    # Time complexity : O(nlogn)
    # Space Complexity : O(1)
	def wave(self, A):
        A.sort()
        for i in range(1,len(A),2):
            A[i],A[i-1]=A[i-1],A[i]
        return A
            

File: Array/WaveArray/WaveArray.go
/**
 * @input A : Integer array
 * 
 * @Output Integer array.
 */
 import "sort"
 func wave(A []int )  ([]int) {
	 sort.Slice(A, func(i, j int) bool {  
		 return A[i] < A[j] 
	 }) 
	 for i:=1;i<len(A);i+=2{
		 A[i],A[i-1]=A[i-1],A[i]
	 }
	 return A
 }
 